src/presentation/webapp/core/pages/sync-rules-list/SyncRulesListPage.tsx

```diff
@@ -336,7 +336,11 @@ export const SyncRulesListPage: React.FC = () => {
             const synchronize = async () => {
                 for await (const { message, syncReport, done } of sync.execute()) {
                     if (message) loading.show(true, message);
-                    if (syncReport) await compositionRoot.reports.save(syncReport);
+                    if (syncReport) {
```

Comment:
This change would not be required if we follow the updateLastExecuted flow as mentioned in previous comment.  Let me know if there is something I am missing.